<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrugGroupBox</class>
 <widget class="QGroupBox" name="DrugGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>339</height>
   </rect>
  </property>
  <property name="title">
   <string notr="true">Medication Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="medicationTitle">
       <property name="toolTip">
        <string notr="true">Name of the medication</string>
       </property>
       <property name="text">
        <string notr="true">Medication:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="medicationLabel">
       <property name="toolTip">
        <string notr="true">Name of the medication</string>
       </property>
       <property name="text">
        <string notr="true">PH MEDICATION</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="genericTitle">
       <property name="toolTip">
        <string notr="true">Generic</string>
       </property>
       <property name="text">
        <string notr="true">Generic:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="genericLabel">
       <property name="toolTip">
        <string notr="true">Generic</string>
       </property>
       <property name="text">
        <string notr="true">PH GENERIC</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="manufacturerTitle">
       <property name="toolTip">
        <string notr="true">Manufacturer</string>
       </property>
       <property name="text">
        <string notr="true">Manufacturer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="manufacturerLabel">
       <property name="toolTip">
        <string notr="true">Manufacturer</string>
       </property>
       <property name="text">
        <string notr="true">PH MANUFACTURER</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="ndcTitle">
       <property name="toolTip">
        <string notr="true">National Drug Code</string>
       </property>
       <property name="text">
        <string comment="NDC DISPLAY LABEL">NDC:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ndcLabel">
       <property name="toolTip">
        <string notr="true">National Drug Code</string>
       </property>
       <property name="text">
        <string notr="true">PH NDC</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="formTitle">
       <property name="toolTip">
        <string notr="true">Medication form</string>
       </property>
       <property name="text">
        <string notr="true">Form:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="formLabel">
       <property name="toolTip">
        <string notr="true">Medication form</string>
       </property>
       <property name="text">
        <string notr="true">PH FORM</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="strengthTitle">
       <property name="toolTip">
        <string notr="true">Strength</string>
       </property>
       <property name="text">
        <string notr="true">Strength:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="strengthLabel">
       <property name="toolTip">
        <string notr="true">Strength</string>
       </property>
       <property name="text">
        <string notr="true">PH STRENGTH</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="dispenseTitle">
       <property name="toolTip">
        <string notr="true">Dispense units can be anything: bottles, capsules, etc.</string>
       </property>
       <property name="text">
        <string comment="DISPENSE UNITS DISPLAY LABEL">Dispense units:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dispenseLabel">
       <property name="toolTip">
        <string notr="true">Dispense units can be anything: bottles, capsules, etc.</string>
       </property>
       <property name="text">
        <string notr="true">PH DISPENSE UNITS</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="unitTitle">
       <property name="toolTip">
        <string notr="true">Unit size (optional)</string>
       </property>
       <property name="text">
        <string notr="true">Unit size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="unitLabel">
       <property name="toolTip">
        <string notr="true">Unit size (optional)</string>
       </property>
       <property name="text">
        <string notr="true">PH UNIT SIZE</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="instructionsTitle">
       <property name="toolTip">
        <string notr="true">Default instructions for prescriptions using this medication</string>
       </property>
       <property name="text">
        <string notr="true">Default instructions:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="instructionsLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string notr="true">Default instructions for prescriptions using this medication</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string notr="true">PH INSTRUCTIONS</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="activeTitle">
       <property name="toolTip">
        <string notr="true">Active or inactive status</string>
       </property>
       <property name="text">
        <string notr="true">Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="activeLabel">
       <property name="toolTip">
        <string notr="true">Active or inactive status</string>
       </property>
       <property name="text">
        <string notr="true">PH ACTIVE</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
